tests/testit/test-hooks-md.R

```diff
@@ -67,8 +67,10 @@ x = "1.png"
 w = h = 1
 ex = "style='margin: 0;'"
 cap = "foo"
-opt <- function(w = NULL, h =NULL, ex = NULL, cap = NULL, show = 'asis', ...) {
-  list(out.width = w, out.height = h, out.extra = ex, fig.cap = cap, fig.show = show, ...)
+opt <- function(w = NULL, h =NULL, ex = NULL, cap = NULL, show = 'asis',
+                fig.align = 'default', ...) {
+  list(out.width = w, out.height = h, out.extra = ex,
+       fig.cap = cap, fig.show = show, fig.align = fig.align, ...)
```

Comment:
```suggestion
```suggestion
opt = function(
  w = NULL, h = NULL, ex = NULL, cap = NULL, show = 'asis',
  fig.align = 'default', ...
) {
  list(
    out.width = w, out.height = h, out.extra = ex,
    fig.cap = cap, fig.show = show, fig.align = fig.align, ...
  )
```